cuplaConfig.cmake

```diff
@@ -273,7 +273,7 @@ endif()
 # Find cupla version.
 ################################################################################
 # FIXME: Add a version.hpp
```

Comment:
This is fixed, maybe remove the line?

Reply:
In cupla we have currently no version.hpp file

Reply:
It is linked above xd

Reply:
ohh thanks so I need to change the version there too.

Reply:
Yes, as I wrote in the review above.